docs/technology_models/grid.md

```diff
@@ -0,0 +1,70 @@
+# Grid Performance and Cost Models
```

Comment:
Very nice job on this page. It holds a lot of information in a very clear and concise way. Thank you!

Reply:
Yaaay thank you @kbrunik!